specifyweb/frontend/js_src/lib/components/FormEditor/createView.ts

```diff
@@ -263,38 +263,56 @@ function createViewFromTemplate(
   const originalNames = filterArray(
     view.altViews.altViews.map(({ viewDef }) => viewDef)
   );
-  const nameMapper = Object.fromEntries(
-    originalNames.map((name) => [name, getUniqueDefinitionName(name, viewSets)])
-  );
+
+  const uniqueName = getUniqueDefinitionName(name, viewSets);
```

Comment:
the name coming from the user was supposed to only be used for the `view` name (as that is what they would specify in the `<subview view="some name here">`
all the other places specify should generate a name of it's own, without relying on what the user provided